optuna/storages/_rdb/alembic/versions/v3.0.0.a.py

```diff
@@ -8,6 +8,7 @@
 from typing import Any
 from typing import List
 
+import sqlalchemy
```

Comment:
Not a strong opinion, but `import sqlalchemy as sa` is often used in optuna.

Reply:
Thanks. It sounds consistent. [81a2e9f](https://github.com/optuna/optuna/pull/3917/commits/81a2e9f00583a018e4e49444b8bcdcbd6f19d0ef)